src/lib.rs

```diff
@@ -60,13 +60,18 @@ impl PixelFormat {
     }
 }
 
+// See https://aomediacodec.github.io/av1-spec/#color-config-semantics
+// and https://en.wikipedia.org/wiki/Chroma_subsampling#Sampling_positions.
 #[repr(C)]
 #[derive(Debug, Default, PartialEq, Copy, Clone)]
 pub enum ChromaSamplePosition {
     #[default]
-    Unknown = 0,
-    Vertical = 1,
-    Colocated = 2,
+    Unknown = 0, // Corresponds to AV1's CSP_UNKNOWN (CENTER assumed).
+    Vertical = 1,  // Corresponds to AV1's CSP_VERTICAL (MPEG-2, also called "left").
+    Colocated = 2, // Corresponds to AV1's CSP_COLOCATED (BT.2020, also called "top-left").
+}
+impl ChromaSamplePosition {
+    const CENTER: ChromaSamplePosition = ChromaSamplePosition::Unknown; // JPEG/"center"
```

Comment:
do we need this exposed in the C API ?

If so we would have to add this as a pub const in `capi/types.rs` similar to some of the `AVIF_COLOR_PRIMARIES_*`.

Reply:
> do we need this exposed in the C API ?

Not sure this is worth it. What do you think?